build/BuildFrameworksTasks/BuildDesktopVKTask.cs

```diff
@@ -0,0 +1,34 @@
+﻿
+namespace BuildScripts;
+
+[TaskName("Build DesktopVK")]
+public sealed class BuildDesktopVKTask : FrostingTask<BuildContext>
+{
+    // TEMP: Until OSX and Linux is setup to work.
+    public override bool ShouldRun(BuildContext context) => context.IsRunningOnWindows();
+
+    public override void Run(BuildContext context)
+    {
+        // TODO: This should be moved to its own seperate
+        // build process so that it doesn't run on all MG builds.
+        {
+            var buildDir = "src/monogame/external/sdl2/sdl/build";
+            context.CreateDirectory(buildDir);
+            context.StartProcess("cmake", new ProcessSettings { WorkingDirectory = buildDir, Arguments = "-A x64 -D CMAKE_MSVC_RUNTIME_LIBRARY=MultiThreaded ../" });
```

Comment:
Should we be also building for windows arm? `-A ARM` ?

Reply:
I'm really super skeptical with win-arm64, it has an extremely low adoption rate (below 0.01% if we look at the Steam stats), and I feel like it's just going to be extra work.